modules/flann/include/opencv2/flann/lsh_table.h

```diff
@@ -163,7 +163,7 @@ class LshTable
         feature_size_ = feature_size;
         CV_UNUSED(key_size);
         std::cerr << "LSH is not implemented for that type" << std::endl;
-        assert(0);
+        CV_Assert(0 && "LSH is not implemented for that type");
```

Comment:
It would be better to remove `std::cerr` and use `CV_Error` in this case:
```.cpp
CV_Error(Error::StsNotImplemented, "LSH is not implemented for that type" );
```